VK/VK/Presentation/News/Cells/PostAndImageTableViewCell.swift

```diff
@@ -61,7 +61,7 @@ extension PostAndImageTableViewCell: UICollectionViewDataSource {
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
         1
     }
-    
+
     func collectionView(
```

Comment:
```suggestion
    func collectionView(
```
тоже в Public Methods